lsp/src/header.ml

```diff
@@ -0,0 +1,91 @@
+open Import
+
+(* TODO rewrite all this using faraday/angstrom *)
```

Comment:
you can take code from https://github.com/c-cube/jsonrpc2/blob/master/src/jsonrpc2_core.ml#L515 if you want. Parsing these  is pretty simple.

Reply:
Would you mind sending a PR?